include/swift/AST/SemanticAttrs.def

```diff
@@ -0,0 +1,79 @@
+//===--- Semantics.def - Semantics Attribute Definitions -------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This macro is used to map the global definition name of a semantic
+// attribute to its raw value.
+//   NAME: the global name used in the compiler
+//   C_STR: the raw value used in swift
+//
+// SEMA_ATTR(NAME, C_STR)
+//
+//===----------------------------------------------------------------------===//
+
+SEMA_ATTR(STRING_EQUALS, "string.equals")
```

Comment:
Why not use SEMANTICS_ATTR? Then it is clear that you are not referring to sema. But maybe I am being too worried.

Reply:
Also, I would suggest maybe adding a check with a nice error if SEMA_ATTR is not defined and you included the function.